packages/cli/generators/relation/utils.generator.js

```diff
@@ -15,6 +15,20 @@ exports.relationType = {
   hasMany: 'hasMany',
 };
 
+class AstLoopBackProject extends ast.Project {
+  constructor(options = {}) {
```

Comment:
AFAICT, the `options` argument is not used, please remove it.

Reply:
Fixed.